test/parallel/test-next-tick-errors.js

```diff
@@ -20,6 +20,28 @@ process.nextTick(function() {
   order.push('C');
 });
 
+assert.throws(
```

Comment:
If you want to test the "same" thing as `process.nextTick()`, you could add `testNextTickWith()`.